tgmc.dme

```diff
@@ -12,6 +12,7 @@
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
 #include "_maps\map_files\generic\Admin_Level.dmm"
+#include "_maps\map_files\Pillar_of_Spring\TGS_Pillar_of_Spring.dmm"
```

Comment:
like the Theseus this shouldn't be included

Reply:
Ok i took it out